### DISCUSSION TITLE
Unconfirmed On Chain Balance

### DISCUSSION THREAD

**Discussion by MilleniumFaclon** [Ride-The-Lightning/RTL#1190] [Troubleshooting Questions] [answered]
Hi There,

I am a complete noob with LND network, I have run up RaspiBlitz 1.7.2 on Raspberry Pi 4 Model B Rev 1.4.

My goal is to establish a channel with some peers and strengthen the network by routing payments through my node.

Unfortunately, during my discovery phase I find myself with all of my on-chain balance 'unconfirmed' with two UTXO's which total my entire balance.

I want to fund my lightning wallet so I can open channels to peers but my balance is unconfirmed and has been unconfirmed for months I suspect due to many botched attempts to move funds between wallets, probably with insufficient fees.  

Please see a record of my transactions to date, and the UTXO's. I note that in the transaction details there are some negative values for block confirmation i.e. -9,886 this I do not understand.

I have read through the RTL doco and am completely lost, please help! If anyone can guide me out of this mess, and help me to position myself so that I can connect to peers, route payments and be productive I would be eternally grateful (:

Thanks

MF
![negBlockConf](https://user-images.githubusercontent.com/117576155/218294456-451ed44b-fb5f-4062-9adf-8d9ecf9ecf66.png)
![Utxos](https://user-images.githubusercontent.com/117576155/218294458-9795b3a4-76ff-4696-bb7f-670e526ed1ae.png)
![UTXO2](https://user-images.githubusercontent.com/117576155/218294459-71e8685a-747a-496e-8241-5d6103a2a1be.png)
![utxo1](https://user-images.githubusercontent.com/117576155/218294461-d3242c52-afa7-4833-baa7-93023d1ebdb0.png)
![Screenshot from 2023-02-12 14-41-23](https://user-images.githubusercontent.com/117576155/218294462-5100c50d-2273-42d3-bebe-175f49d88efd.png)

**Comment by saubyk** [comment]
Hi @MilleniumFaclon can you please confirm the LND version you're on?

Also please inspect the transactions on any bitcoin explorer e.g. mempool.space to confirm if those are actually settled. You should be able to search with your transaction hash

**Reply by MilleniumFaclon** [reply]
Thanks saubyk,

I am on lnd version 0.14.2-beta commit=v0.14.2-beta

I checked mempool for the transaction tying up my balance, I'm  confused as there does appear to be block confirmations, if that's what you mean by settled.
![mempool_trx](https://user-images.githubusercontent.com/117576155/218398285-f9a7a9e9-1890-419a-b69c-f15b2756d741.png)


Completely lost, any helpful suggestions welcomed.

Thanks  

MF

**Comment by saubyk** [comment, marked as answer]
Hi @MilleniumFaclon please update your LND version to 0.15.5 asap, that should address your problem

**Comment by MilleniumFaclon** [comment]
Thanks saubyk,

I have upgraded successfully to lnd version 0.15.4-beta commit=v0.15.4-beta and the problem persists unfortunately.  All on chain balance unconfirmed.  Suggestions as to how I can overcome my liquidity issue most welcome.

Cheers

MF

**Comment by MilleniumFaclon** [comment]
Hi @saubyk 

I just realised I in fact hadn't updated to the latest version.  That last patch (0.15.5) has solved my issue and my balance is recovered!!
I can't thank you enough sir

Cheers

MF
